packages/tocco-ui/src/Select/IndicatorsContainer.js

```diff
@@ -16,11 +16,12 @@ const IndicatorsContainer = props => {
     <components.IndicatorsContainer {...restProps}>
       {children}
       {openAdvancedSearch
+      && !readOnly
       && <span
         onTouchEnd={e => e.stopPropagation()}
         onMouseDown={e => e.stopPropagation()}>
         <Button
-          disabled={readOnly || !openAdvancedSearch}
+          disabled={!openAdvancedSearch}
```

Comment:
select fields never have an advanced search button but also use the select component (as remote fields).  If no "openadvancedSearch" is defined the icon should not be displayed.

Reply:
see 83b567c3cf24252169282ef0b3bac6882ba198d9